examples/drivers/windows/otLwf/filter.h

```diff
@@ -111,6 +111,29 @@ typedef enum OTLWF_DEVICE_STATUS
 #define OT_EVENT_TIMER_RUNNING      1
 #define OT_EVENT_TIMER_FIRED        2
 
+#if OPENTHREAD_CONFIG_PLATFORM_MESSAGE_MANAGEMENT
+
+typedef struct BufferPool
+{
+    struct BufferPool* Next;
+    uint8_t Buffers[0];
+
+} BufferPool;
+
+enum
+{
+    kPageSize                = PAGE_SIZE,
+    kPagesPerBufferPool      = 1,
```

Comment:
The number of buffers to allocate per instance largely depends on the amount of buffering you want for MTD children.  For non-sleepy MTDs, a router needs to reassemble IPv6 datagrams before delivering them to the MTD.  The more important issue to consider is that for sleepy MTDs, the router must buffer messages until the MTD polls for them.  The poll period is determined by the MTD and could be minutes or more.  To first order, I would size the buffer pool based on the number of sleepy children you wish to support and the amount of buffer per sleepy child.